src/content/docs/extensions/attach/rdbms.mdx

```diff
@@ -319,7 +353,56 @@ The below table lists some common connection string parameters:
 | `password`  | Postgres password | [empty] |
 | `port`      | Port number | 5432 |
 
-#### 3. Scan from PostgreSQL tables
+#### 3. Data type mapping from PostgreSQL to kuzu
```

Comment:
Can you double check we unify `kuzu` and `Kuzu` to `Kùzu` in the file?

Reply:
done